talkideas.md

```diff
@@ -8,3 +8,39 @@ In-depth debugger talk
 ======================
 *TODO: Rewrite*
 
+
+Developing Developers
+=====================
+Software development is a young profession. Our career paths are still geared up to accept junior developers from academia, make them productive in a team, and then preparing them for a career in management. Technical skills and experience are devalued in preference of line management and project management.
+
+We have a responsibility for developing the technical skills and experience of developers, for building a better profession and for creating a healthy environment to work in. How can we shape careers that challenge and value developers, and how do you identify and respond when things go wrong?
+
+This talk will use my insights from building career paths from scratch at emerging companies, and examine the successes and failures that are common across the industry.
+
```

Comment:
I really like this one. Aside from maybe a few grammar nitpicks, I think it's solid. (Could be really good for http://2016.theleaddeveloper.com/)